### PR TITLE
Staged tree doc

### PR DESCRIPTION
**Pull request by g-walley**
Staged tree documentation for the sphinx readthedocs page

### CODE REVIEW

File: docs/quickstart/quickstart_staged_tree_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Image\n",
+    "g = st.dot_graph()\n",
+    "g.set('ranksep', 1)\n",
+    "g.set('nodesep', 0.005)\n",
+    "g.get_edge(('s8', 's20'))[-1].set_style('dashed')\n",
```
Comment (g-walley) [inline docs/quickstart/quickstart_staged_tree_notebook.ipynb:161]: This is really cool! 

One issue I do have is that it is not clear what this is doing unless you dig into the api.

Something like this (although much more verbose) would be clearer to the reader.
```python
for edge in g.get_edge(src_or_list='s8', dst='s20'):
    if "Hard" in edge.get("label"):
        edge.set_style('dashed')
```

This makes it clear that `get_edge` it is returning a list, and that you intend to modify that specific edge.

File: docs/quickstart/quickstart_staged_tree_notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    "probability of an individual, who has arrived at the situation, going along one of its edges. In order to create a CEG, we first need to elict a *staged tree*. \n",
+    "This is done by first need to partition situations into *stages*, which are collection of situation in the event tree whose immediate evolutions, i.e. their\n",
+    "associated conditional transition parameter vectors, are equivalent. To indicate this symmetry, all situations in the same stage are assinged a single colour.\n",
+    "\n",
+    "Identification of the stages in the event tree can be done using any suitable model selection algorithm. Currently, the only avaible selection algoirhtm in `cegpy` is the *agglomerative hierarchical clustering (AHC)* algorithm (Freeman and Smith, 2011).\n",
+    "\n",
+    "In order to creat a staged tree in `cegpy` we first initialize a `StagedTree` object from the dataset and then run th AHC algorithm using the `create_AHC_tarnsitions` method, as displayed below. The output of the AHC algorithm is a dictionary containing the following information:\n",
```
Comment (g-walley) [inline docs/quickstart/quickstart_staged_tree_notebook.ipynb]: Minor spelling: "th AHC".

When I was trying to get the docs to build, I went through and cleaned up some other spelling mistakes. I missed this one.

FYI I use "Code Spell Checker" in vscode (Identifier: streetsidesoftware.code-spell-checker) and you may also find this useful.

File: docs/quickstart/quickstart_staged_tree_notebook.ipynb (hunk 1 above)
Comment (Kaasiak) [reply docs/quickstart/quickstart_staged_tree_notebook.ipynb]: Oh yes I agree!

File: docs/quickstart/quickstart_staged_tree_notebook.ipynb (hunk 2 above)
Comment (Kaasiak) [reply docs/quickstart/quickstart_staged_tree_notebook.ipynb]: Thanks! I was meant to add spell checking to my workspace ages ago :) that is very helpful.

**Comment by Kaasiak** [comment]
The StagedTree user guide is I think mostly complete. 

I moved the visual modification section to a separate notebook because it applies not only to staged trees but also all other objects.

Apologies for the forced push. I didn't notice the previous changes made on the branch and then messed merging ...

**Pull request merged**